src/OpenRiaServices.Server/Framework/Data/DomainOperationEntry.cs

```diff
@@ -90,7 +90,7 @@ protected DomainOperationEntry(Type domainServiceType, string name, DomainOperat
         /// </summary>
         /// <param name="name">A name.</param>
         /// <returns>name, but without "Async" at the end</returns>
-        private static string RemoveAsyncFromName(string name)
+        public static string RemoveAsyncFromName(string name)
```

Comment:
I don't think we want to make this API public.

Maybe we can update DomainServiceDescription so that `GetInvokeOperation` and related methods works with "Async" as an alternative

other alternatives are internal or copy implementation

Reply:
Looked into different alternatives and decided to go for a copy implementation.

- For internals we would need InternalsVisibleTo
- Looked into moving it into DomainServiceDescription but could not find a good solution